### DISCUSSION TITLE
Country area to Province/State

### DISCUSSION THREAD

**Discussion by ghost** [saleor/saleor#9930] [General] [answered]
The current address information uses the country area. Has the saleor team considered changing it to a province/state?

**Comment by karolkielecki** [comment]
Hi @chrislais, we haven't considered changing it. Why would you like to change it? I've passed on your suggestion to the team.

**Reply by ghost** [reply]
Hi @karolkielecki, since it looks like the country area is not commonly used on websites, I think most people don't know how to fill it out.

**Comment by patrys** [comment, marked as answer]
Are you asking about the API or the UI? API-wise, state, province, region, oblast, voivodeship, and district are all types of country administrative areas.

In terms of UI, Saleor provides an address validation endpoint that tells you what labels to use for individual address fields in a particular country:

```graphql
{
  addressValidationRules(countryCode: US) {
    countryAreaType
    cityType
    cityAreaType
  }
}
```
